bucketeer2/src/main/kotlin/io/bucketeer/sdk/android/BKTClientImpl.kt

```diff
@@ -173,4 +150,44 @@ internal class BKTClientImpl(
 
     return raw.getVariationValue(defaultValue)
   }
+
+  @MainThread
+  private fun scheduleBackgroundTasks() {
+    ProcessLifecycleOwner.get().lifecycle.addObserver(TaskScheduler(component, executor))
+  }
+
+  companion object {
+    internal fun fetchEvaluationsSync(
```

Comment:
I moved `fetchEvaluationsSync` to static, so that we can use this from polling tasks.